docs/en/pid-data-model.rst

```diff
@@ -0,0 +1,393 @@
+
+.. include:: ../common/common_definitions.rst
+
+.. _pid_data_model.rst:
+
+PID Data Model
+++++++++++++++
+
+The Person Identification Data (PID) is issued by the PID Provider following national laws and allows a natural person to be authenitcated and identified. 
+
+The User attributes carried in the Italian PID are:
+
+    - Current Family Name
+    - Current First Name
+    - Date of Birth
+    - Place of Birth
+    - Unique Identifier
+    - Taxpayer identification number
+
+The italian PID is extended according to the `OpenID Identity Assurance Profile [OIDC.IDA] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html>`_, that enables the binding of the PID to a national trust framework, giving all the evidence of the identity proofing procedures underlying the PID issuing in both remote and proximity flows.
+
+The PID data format and the mechanism through which it is issued into the Wallet Instance and presented to a RP will be detailed in the next sections. 
```

Comment:
editorial, we use acronyms like RP here and we need to have these in the defined term, acronyms section